src/dwi/tractography/seeding/seeding.cpp

```diff
@@ -157,6 +167,22 @@ namespace MR
           }
         }
 
+        opt = get_options ("seed_coordinates_fixed"); 
+        for (size_t i = 0; i < opt.size(); ++i) {          
+          Coordinates_fixed* seed = new Coordinates_fixed (str(opt[i][0]), opt[i][1]);
+          list.add (seed);
+        }
+
+        // those exclusive of other seeding mechanisms go to the end, with dynamic seeder last
+
+        opt = get_options ("seed_coordinates_global");
+        if (opt.size()) { 
+          if (list.num_seeds())
+            throw Exception ("If seeding from coordinates to a global target, cannot specify any other type of seed!");
```

Comment:
"Global" might be misinterpreted here. Maybe:
"If seeding from pre-specified coordinates with no fixed number of streamlines per coordinate, ..."?